combiners/SSNet_predictions_notebook.py

```diff
@@ -0,0 +1,553 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # SSNet Predictions
```

Comment:
Please use data folder for all the csv files. You can update the path (or name) in SSNet_predictions.py directly

Reply:
In the new PR I won't include the csv files and will try to make sure all the scripts use the same file paths. I had some other problems with the model training/testing scripts I discussed with @atvassilev, but I will open separate issues for those with more detail.